tests/lax_scipy_sparse_test.py

```diff
@@ -218,43 +225,46 @@ def test_cg_weak_types(self):
     shape=[(5, 5)],
     dtype=[np.float64, np.complex128],
     preconditioner=[None, 'identity', 'exact', 'random'],
+    symmetric=[False, True]
   )
   def test_bicgstab_against_scipy(
-      self, shape, dtype, preconditioner):
+    self, shape, dtype, preconditioner, symmetric):
```

Comment:
I find this indentation change pretty confusing, because now the argument continuation is not visually distinguished from the content of the block. Revert the indentation change please